backend/src/main/java/com/woowacourse/levellog/prequestion/domain/PreQuestion.java

```diff
@@ -0,0 +1,60 @@
+package com.woowacourse.levellog.prequestion.domain;
+
+import com.woowacourse.levellog.common.domain.BaseEntity;
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class PreQuestion extends BaseEntity {
+
+    @ManyToOne
+    @JoinColumn(nullable = false, foreignKey = @ForeignKey(name = "fk_pre_question_levellog"))
+    private Levellog levellog;
+
+    @ManyToOne
+    @JoinColumn(nullable = false, foreignKey = @ForeignKey(name = "fk_pre_question_from_member"))
+    private Member from;
+
+    @Lob
+    private String preQuestion;
+
+    public PreQuestion(final Levellog levellog, final Member from, final String preQuestion) {
+        validateIsAuthor(levellog, from);
+        validateQuestion(preQuestion);
+
+        this.levellog = levellog;
+        this.from = from;
+        this.preQuestion = preQuestion;
+    }
+
+    private void validateIsAuthor(final Levellog author, final Member member) {
+        if (author.isAuthor(member)) {
+            throw new InvalidFieldException("내 글의 사전 질문을 작성할 수 없습니다.");
```

Comment:
```suggestion
            throw new InvalidFieldException("내 레벨로그에 사전 질문을 작성할 수 없습니다.");
```
헷갈릴 우려가 있습니다. 정확한 도메인 이름을 사용해주세요! 🤤

Reply:
도메인 이름을 쓰는 방식으로 수정했습니당